servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/RetryingHttpRequesterFilterTest.java

```diff
@@ -194,8 +199,12 @@ public Single<StreamingHttpResponse> request(final StreamingHttpRequest request)
         HttpResponseException e = assertThrows(HttpResponseException.class,
                 () -> normalClient.request(normalClient.get("/")));
         assertThat("Unexpected exception.", e, instanceOf(HttpResponseException.class));
-        assertThat("Unexpected calls to select.", lbSelectInvoked.get(), is(4));
-        assertThat("Response payload body was not drained on every mapping", responseDrained.get(), is(4));
```

Comment:
test was racy in that some times LB would not be ready and only 3 requests were actually sent, which triggered https://github.com/apple/servicetalk/issues/2405